content/en/docs/guides/certificate_management/certificate-rotation.md

```diff
@@ -0,0 +1,206 @@
+   kubectl rollout restart deploy osm-injector -n $osm_namespace
+   kubectl rollout restart deploy osm-bootstrap -n $osm_namespace
+   ```
+
+When the components gets re-deployed, you should be able to eventually see the new `osm-ca-bundle` secret in `$osm_namespace`:
+
+```console
+kubectl get secrets -n $osm_namespace
+```
+
+```
+NAME                           TYPE                                  DATA   AGE
+osm-ca-bundle                  Opaque                                3      74m
+```
+
+The new expiration date can be found with the following command:
+
+```console
+kubectl get secret -n $osm_namespace $osm_ca_bundle -o jsonpath='{.data.ca\.crt}' |
+    base64 -d |
+    openssl x509 -noout -dates
+```
+
+For Envoy services and validation certificates to be rotated the data plane components must be restarted.
+
+#### Other certificate providers
+The process of rotating the root certificate will be different for certificate providers other than Tresor. For Hashicorp Vault and cert-manager.io, users will need to rotate the root certificate themselves outside of OSM.
+
+### Certificate Rotation with MeshRootCertificate
+
+>  WARNING:  This feature is currently in preview and requires the `EnableMeshRootCertificate` feature flag enabled in the MeshConfig.  
+
+The `MeshRootCertificate` is introduced in OSM v1.3. This custom resource can be enabled by using the `EnableMeshRootCertificate` feature flag when deploying OSM.
+
+```console
+osm install --set="osm.featureFlags.enableMeshRootCertificate=true"
+```
+
+> Important: If OSM is installed with `EnableMeshRootCertificate` disabled, but the feature flag is then later enabled in the MeshConfig, the control plane components must be restarted. This allows the flag to be picked up and begin using the `MeshRootCertificate` for certificate management. There will be no change in the root certificate, but it will now be managed by the `MeshRootCertificate`. 
+
+### MeshRootCertificate
+
+Each `MeshRootCertificate` represents a certificate that can be used in the mesh. There can be only two `MeshRootCertificate`'s that are used in the mesh at a given time and there must always be one `MeshRootCertificate` in the **active** `role`.
+
+The following are the possible `MeshRootCertificate role`s:
+
+- `active`: The certificate is used to sign **and** validate certificates in the mesh. There must always be one active certificate.
+- `passive`: The certificate is used to validate certificates in the mesh.
+- `inactive`: The certificate is not used in the mesh.
+
+The following is an example of the `MeshRootCertificate` that will be created. The `provider` field can be `tresor`, `certManager`, or `vault`.  Before using the providers for [CertManager](certificates.md#using-cert-manager) or [Vault](certificates.md#using-hashicorp-vault), additional pre-configuration steps are required.  Refer to the [API documentation](#TODO) on how to configure the `MeshRootCertificate` fields for each provider.
```

Comment:
I think this part should be put after the yaml so the user can see what the yaml looks like first
"The `provider` field can be `tresor`, `certManager`, or `vault`.  Before using the providers for [CertManager](certificates.md#using-cert-manager) or [Vault](certificates.md#using-hashicorp-vault), additional pre-configuration steps are required.  Refer to the [API documentation](#TODO) on how to configure the `MeshRootCertificate` fields for each provider."